etrago/appl.py

```diff
@@ -121,17 +121,19 @@
     },
     "network_clustering_ehv": False,  # clustering of HV buses to EHV buses.
     "disaggregation": "uniform",  # None, 'mini' or 'uniform'
+    # Temporal Complexity:
     "snapshot_clustering": {
-        "active": False,  # choose if clustering is activated
+        "active": True,  # choose if clustering is activated
```

Comment:
The default settings are now clustering the first two hours of the year to 5 days and skip every 3rd hour. Maybe we should change this a bit.

Reply:
Thank you!